<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionsForm</class>
 <widget class="QWidget" name="TransactionsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidgetTransactions">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border: none;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>400</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrevious">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Eteenp√§in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrevious_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Taaksepain</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetTransactions_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>471</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <zorder>listWidgetTransactions_2</zorder>
  <zorder>listWidgetTransactions</zorder>
  <zorder>btnStop</zorder>
  <zorder>btnNext</zorder>
  <zorder>btnPrevious</zorder>
  <zorder>btnNext_2</zorder>
  <zorder>btnPrevious_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
